lib/src/config_util.cc

```diff
@@ -0,0 +1,29 @@
+#include <internal/config_util.hpp>
+#include <cctype>
+
+namespace hocon {
+
+    bool is_whitespace(char c) {
+        switch (c) {
+            case ' ':
+            case '\n':
+            // TODO: these complain about char overflow, what do.
```

Comment:
Probably; we need to support the same input. Mostly that doesn't require extra work on our part - we just read them into a string as bytes - but if we're manipulating or comparing against the strings we need to do extra work.